cypress.config.ts

```diff
@@ -6,6 +6,6 @@ export default defineConfig({
     setupNodeEvents() {
       // implement node event listeners here
     },
-    baseUrl: 'http://localhost:3000',
+    baseUrl: 'http://localhost:8080',
```

Comment:
very small suggestion: poodle uses `:8080` too - could be worth picking a different port if it's easy to do w/ no headaches. even the slightest whisper of a headache, would abandon my suggestion immediately

Reply:
no whispers! updated to port `1234` 🚀